src/main/java/ladder/domain/GenerateLineMethod.java

```diff
@@ -0,0 +1,23 @@
+package ladder.domain;
+
+import java.util.Random;
+import java.util.function.Supplier;
+
+public class GenerateLineMethod implements Supplier {
+    private Boolean ladder;
```

Comment:
```suggestion
    private boolean ladder;
```
특별한 경우가 아닌 이상 항상 원시타입을 사용해주세요!
참고자료: https://velog.io/@yundleyundle/Item6-%EB%B6%88%ED%95%84%EC%9A%94%ED%95%9C-%EA%B0%9D%EC%B2%B4-%EC%83%9D%EC%84%B1%EC%9D%84-%ED%94%BC%ED%95%98%EB%9D%BC

Reply:
아 네 알겠습니다.